@xen-orchestra/web-core/lib/components/network/VtsNetworkStatus.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <UiInfo class="status-indicator text-ellipsis" :accent="computedStatusProps.accent">
+    <p v-tooltip class="text-ellipsis" :class="{ card }">{{ computedStatusProps.text }}</p>
```

Comment:
What the point to use balise `p` here ? maybe we can use a span instead
```suggestion
    <span v-tooltip class="text-ellipsis" :class="{ card }">{{ computedStatusProps.text }}</span>
```